app-containers/docker/autobuild/build

```diff
@@ -45,7 +62,6 @@ export BUILDTAGS="pkcs11"
 export GOBUILDTAGS="${BUILDTAGS}"
 
 abinfo "docker-cli: Building ..."
-. ./scripts/build/.variables
```

Comment:
After this script invocation been removed, $GO_BUILDTAGS goes undefined.

Reply:
$GO_BUILDTAGS only when `CGO_ENABLED=1` has data, but we build it not used this variable, so I remove it